### PR TITLE
Fix license info accordian

### LINKED ISSUE #1839: Fix LicenseInfo component
**Describe the bug**
The LicenseInfo component has an bug and therefore is currently disabled in https://github.com/Plant-for-the-Planet-org/treecounter-app/blob/9e6754f790c08461594674883edf8492659ebdf5/app/components/AboutUs/index.native.js#L58 and replaced with a link to https://github.com/Plant-for-the-Planet-org/treecounter-app/blob/9e6754f790c08461594674883edf8492659ebdf5/app/components/AboutUs/index.native.js#L56

**Is it on a Production or Development Build?**
production and develop

**To Reproduce**
1. Go to 'About us'
2. Click on 'Open-Source-License'
3. See error

```
TypeError: undefined is not an object (evaluating 'activeSections.includes')
This error is located at:
    in Accordion (at LicenseInfoList.native.js:70)


```

### PR DESCRIPTION
**Pull request by niteshgrg**
Fixes #1839

Changes in this pull request:
- Supply new mandatory components to accordion.

### CODE REVIEW

File: app/components/AboutUs/index.native.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ import i18n from '../../locales/i18n';
 import { withNavigation } from 'react-navigation';
 import { updateRoute } from '../../helpers/routerHelper/routerHelper.native';
 // import TabContainer from '../../containers/Menu/TabContainer';
-//const LicenseInfo = require('./LicenseInfo.json');
+const LicenseInfo = require('./LicenseInfo.json');
```
Comment (norbertschuler) [inline app/components/AboutUs/index.native.js:10]: I am afraid this file is already 5 month old now: https://github.com/Plant-for-the-Planet-org/treecounter-app/blob/develop/app/components/AboutUs/LicenseInfo.json - so all the information in this file is outdated and need to get updated before we can activate it again.

Comment (norbertschuler) [reply app/components/AboutUs/index.native.js:10]: Updated the licenses. Unfortunately the command `./node_modules/license-checker/bin/license-checker --production --relativeLicensePath --direct --json > license.json` does not work so well as it generates more licenses as just the direct packages we linked in our package.json file for "dependencies". Therefore I extracted the list of packages and converted it into list I could give with the --package parameter as following: 
```
./node_modules/license-checker/bin/license-checker --relativeLicensePath --json --package "@bugsnag/js@6.5.2;@esri/react-arcgis@5.0.0;@react-native-community/async-storage@1.8.1;@react-native-community/cli-platform-android@4.2.2;@react-native-community/cli-platform-ios@4.2.2;@react-native-community/datetimepicker@2.2.2;@react-native-community/geolocation@2.0.2;@react-native-community/masked-view@0.1.6;@react-native-community/netinfo@5.5.1;@react-native-community/viewpager@3.3.0;autosuggest-highlight@3.1.1;axios@0.19.2;bugsnag-react-native@2.23.6;classnames@2.2.6;cookie-cutter@0.2.0;core-js@3.6.4;crypto-js@4.0.0;css-loader@3.4.2;date-fns@2.9.0;deprecated-react-native-listview@0.0.6;dotenv-webpack@1.7.0;esri-loader@2.13.0;formik@2.1.4;history@4.10.1;i18next@19.3.2;intl@1.2.5;jwt-decode@2.2.0;lodash@4.17.15;normalizr@3.6.0;prop-types@15.7.2;react@16.12.0;react-accessible-accordion@2.4.5;react-async-script-loader@0.3.0;react-autosuggest@9.4.3;react-datepicker@2.13.0;react-dom@16.12.0;react-i18next@11.3.3;react-modal@3.11.2;react-native@0.61.5;react-native-check-box@2.1.7;react-native-collapsible@1.5.1;react-native-config@0.12.0;react-native-content-loader@0.1.6;react-native-device-info@5.5.3;react-native-document-picker@3.2.4;react-native-extended-stylesheet@0.12.0;react-native-fs@2.16.6;react-native-gesture-handler@1.6.0;react-native-get-random-values@1.2.0;react-native-google-places-autocomplete@1.4.1;react-native-htmlview@0.15.0;react-native-i18n@2.0.15;react-native-image-carousel@0.4.6;react-native-image-crop-picker@0.28.0;react-native-image-picker@2.3.0;react-native-keyboard-aware-scroll-view@0.9.1;react-native-linear-gradient@2.5.6;react-native-maps@0.26.1;react-native-material-dropdown@0.11.1;react-native-material-textfield@0.16.1;react-native-modal-datetime-picker@8.5.0;react-native-modalbox@2.0.0;react-native-pdf@6.0.1;react-native-permissions@2.0.9;react-native-popover-tooltip@1.1.4;react-native-ratings@6.5.0;react-native-raw-bottom-sheet@2.0.6;react-native-reanimated@1.7.0;react-native-root-toast@3.2.0;react-native-safe-area-context@0.7.3;react-native-screens@2.2.0;react-native-slick@1.0.2;react-native-svg@11.0.1;react-native-svg-uri@1.2.3;react-native-switch@1.5.0;react-native-tab-view@2.13.0;react-native-vector-icons@6.6.0;react-native-webview@8.1.2;react-navigation@4.2.1;react-navigation-drawer@2.4.1;react-navigation-redux-helpers@4.0.1;react-navigation-stack@2.2.1;react-navigation-tabs@2.8.1;react-notifications@1.4.3;react-redux@7.2.0;react-reveal@1.2.2;react-router@5.1.2;react-router-dom@5.1.2;react-select@3.0.8;react-share@4.0.1;react-slick@0.25.2;react-smartbanner@5.1.4;react-stripe-elements@6.1.0;react-tooltip@4.0.3;react-youtube@7.9.0;recaptcha-v3-react@3.0.3;redux@4.0.5;redux-actions@2.6.5;redux-form@8.3.0;redux-logger@3.0.6;redux-thunk@2.3.0;reselect@4.0.0;rn-fetch-blob@0.12.0;rxjs@6.5.4;rxjs-compat@6.5.4;slick-carousel@1.8.1;source-map-explorer@2.3.1;tcomb-form@0.9.21;tcomb-form-native@0.6.20;tcomb-json-schema@0.3.2;uuid@7.0.1;yup@0.28.1" > license.json 
```

**Comment by niteshgrg** [comment]
@norbertschuler Can you redirect me to the new design ?

**Comment by niteshgrg** [comment]
I don't see any new designs for this screen on zeplin. If those can be supplied will change update the complete screen then. @harshvitra

**Comment by norbertschuler** [comment]
> @norbertschuler Can you redirect me to the new design ?

Sorry, I don’t know how to use the new header. Would have to copy from imprint.

**Comment by niteshgrg** [comment]
@norbertschuler Have a look now.

**Pull request merged**